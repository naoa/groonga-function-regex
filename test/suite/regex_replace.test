register functions/regex

table_create Entries TABLE_NO_KEY
column_create Entries title COLUMN_SCALAR ShortText

load --table Entries
[
{"title": "groonga and MySQL"}
]

select Entries --output_columns 'regex_replace(title, " (.*) ", " or ")' --command_version 2
