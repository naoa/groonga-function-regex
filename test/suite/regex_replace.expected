register functions/regex
[[0,0.0,0.0],true]
table_create Entries TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Entries title COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
load --table Entries
[
{"title": "groonga and MySQL"}
]
[[0,0.0,0.0],1]
select Entries --output_columns 'regex_replace(title, " (.*) ", " or ")' --command_version 2
[[0,0.0,0.0],[[[1],[["regex_replace","Object"]],["groonga or MySQL"]]]]
